File: cpp_version/intro/hey_programmer/hey_programmer.cpp
#include <boost/array.hpp>
#include <iostream>
#include <string>

std::string greet(std::string s) {
    return "";
}

int main() {

    std::cout << greet("alvin") << std::endl;             //'hey alvin'
    std::cout << greet("jason") << std::endl;             //'hey jason'
    std::cout << greet("how now brown cow") << std::endl; //'hey how now brown cow'
}

File: python_version/intro/hey_programmer/hey_programmer.py
# write a functions that takes in a param string s and returns a string 'hey s'

def greet(s):
    return f'hey {s}'

if __name__== '__main__':
    print(greet("alvin")) #'hey alvin'
    print(greet("jason")) #'hey jason'
    print(greet("how now brown cow")) #'hey how now brown cow'